inst/rmarkdown/templates/outbreak/skeleton/skeleton.Rmd

```diff
@@ -196,7 +200,9 @@ linelist_cleaned$name <- NULL
 
 
 
-In total there were `r nrow(linelist_cleaned)` cases. There were `r linelist_cleaned %>% filter(sex == "Female") %>% count()` females affected and `r linelist_cleaned %>% filter(sex == "Male") %>% count()` males. 
+In total there were `r nrow(linelist_cleaned)` cases. There were
```

Comment:
maybe we should add to the intro the whole good datapractice of 80 chars per line - but noting that it doesnt actually give a new line in the output text. I.e. double space and return gives a new line

Reply:
I think that may be a good idea since they will encounter the RMD firsthand in RStudio, and the wrapping doesn't really do it justice. The only problem is keeping the inline code chunks contiguous. If they break on a line, then they won't work.

Reply:
@pbkeating